src/utils/flowRight.ts

```diff
@@ -0,0 +1,16 @@
+export default(...funcs: any[]) => {
+  const functions = funcs.reverse();
+  functions.forEach((fnc, i) => {
+    if (typeof fnc !== 'function') {
+      throw new Error(`In "compose" the argument at index "${i}" is not a function.`);
```

Comment:
I don't think we need this extra dynamic type checking, since the functions will fail when we call them below, and TypeScript consumers will be warned about passing non-`Function` arguments (if you accept the change I suggested above).

Reply:
Okay, i'll fix it :)